tests/integration/cli/test_cli.py

```diff
@@ -155,29 +155,41 @@ def test_predict_with_image_path(self, project_path: Path) -> None:
         )
         torch.cuda.empty_cache()
 
-    @pytest.mark.parametrize("export_type", [ExportType.TORCH, ExportType.ONNX, ExportType.OPENVINO])
+    @pytest.mark.parametrize(
+        ("export_type", "compression_type"),
+        [
+            (ExportType.TORCH, None),
+            (ExportType.ONNX, None),
+            (ExportType.OPENVINO, None),
+            (ExportType.OPENVINO, CompressionType.POT),
+            (ExportType.OPENVINO, CompressionType.NNCF),
```

Comment:
@sky0walker99 did you try running the commands via CLI? This is what I tried
```bash
anomalib export --export_type openvino --compression_type NNCF --model Padim --data MVTec --ckpt_path results/Padim/MVTec/bottle/latest/weights/lightning/model.ckpt
```
And this was the error
```bash
│   261 │   │   │   if compression_type and compression_type != CompressionType.FP16:              │
│ ❱ 262 │   │   │   │   model = self._compress_ov_model(model, compression_type, datamodule, met   │
│   263 │   │   │                                                                                  │
│   264 │   │   │   # fp16 compression is enabled by default                                       │
│   265 │   │   │   compress_to_fp16 = compression_type == CompressionType.FP16                    │
│                                                                                                  │
│ /home/ashwin/projects/anomalib_copy/src/anomalib/models/components/base/export_mixin.py:311 in   │
│ _compress_ov_model                                                                               │
│                                                                                                  │
│   308 │   │   │   model = self._accuracy_control_quantization_ov(model, datamodule, metric, ta   │
│   309 │   │   else:                                                                              │
│   310 │   │   │   msg = f"Unrecognized compression type: {compression_type}"                     │
│ ❱ 311 │   │   │   raise ValueError(msg)                                                          │
│   312 │   │                                                                                      │
│   313 │   │   return model                                                                       │
│   314                                                                                            │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
ValueError: Unrecognized compression type: nncf
```

Anyways, the modifications to test are correct but you only need to expose these (the existing ones) 
```python
FP16 = "fp16"
INT8 = "int8"
INT8_PTQ = "int8_ptq"
INT8_ACQ = "int8_acq"
```